client/components/thank-you/index.tsx

```diff
@@ -128,17 +148,23 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 
 	return (
 		<ThankYouContainer className={ classNames( 'thank-you__container', containerClassName ) }>
-			<ThankYouHeader className="thank-you__container-header">
+			<ThankYouHeader className={ classNames( 'thank-you__container-header', headerClassName ) }>
 				{ /* eslint-disable-next-line jsx-a11y/alt-text */ }
 				<img { ...thankYouImage } />
+				<ThankYouFooterContainer>
+					<h1> { thankYouImageFooter } </h1>
+					<h2> { thankYouImageFooterSubtitle } </h2>
+				</ThankYouFooterContainer>
 			</ThankYouHeader>
 			<ThankYouBody>
 				<div>
-					<ThankYouSectionContainer>
-						<ThankYouHeaderTitle className="thank-you__body-header wp-brand-font">
-							{ thankYouTitle }
-						</ThankYouHeaderTitle>
-					</ThankYouSectionContainer>
+					{ thankYouTitle && (
+						<ThankYouSectionContainer>
+							<ThankYouHeaderTitle className="thank-you__body-header wp-brand-font">
+								{ thankYouTitle }
+							</ThankYouHeaderTitle>
+						</ThankYouSectionContainer>
+					) }
```

Comment:
As noted above, let's just move `thankYouTitle` into the blue header. I think we want to move those titles into the header for all use cases.
```suggestion
```